Kp: 400
Kd: 200
limit1: 500
limit2: 1000
limit3: 1000
limit4: 50
limit5: 50
limit6: 50
velocity_limit1: 1
velocity_limit2: 1
velocity_limit3: 1
velocity_limit4: 1
velocity_limit5: 1
velocity_limit6: 1
dt: 0.01
weight_cost_u: 0.001
weight_cost_a: 1
weight_cost_slack: 1
w: 6